src/modules/roc_netio/target_uv/roc_netio/transceiver.cpp

```diff
@@ -214,12 +198,88 @@ void Transceiver::stop_sem_cb_(uv_async_t* handle) {
     roc_panic_if_not(handle);
 
     Transceiver& self = *(Transceiver*)handle->data;
-    self.stop_();
+    self.stop_all_();
     self.close_();
     self.process_tasks_();
 }
 
+void Transceiver::close_cb_(uv_handle_t* handle) {
```

Comment:
I suggest to rename some methods to make it clear what objects they are working with:

* close_cb_() -> close_sem_cb_()
* wait_stopped_() -> wait_elements_stopped_()
* wait_closed_() -> wait_sems_closed_()
* stop_all_() -> stop_all_elements_()
* close_() -> close_sems_()

Reply:
done

Reply:
It seems that close_cb_ was not renamed.

Reply:
One more time.